src/github/entities/user-installation.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class UserInstallation extends Base {
+  @Column({ type: 'varchar', length: 20 })
+  installationId: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  organizationName: string;
+
+  @ManyToOne(() => User, (user) => user.installations)
+  user: User;
+}
```

Comment:
PullProofAI

The `UserInstallation` entity looks well-defined with necessary properties and relationships. It's good practice to have clear and concise entity definitions.

```diff
+ import { Base } from 'src/universal/base.entity';
+ import { Column, Entity, ManyToOne } from 'typeorm';
+ import { User } from './user.entity';
+
+ @Entity()
+ export class UserInstallation extends Base {
+   @Column({ type: 'varchar', length: 20 })
+   installationId: string;
+
+   @Column({ type: 'varchar', length: 255 })
+   organizationName: string;
+
+   @ManyToOne(() => User, (user) => user.installations)
+   user: User;
+ }
```



<!-- This is an auto-generated comment by PullProofAI -->